applications/serial_lte_modem/src/slm_at_host.c

```diff
@@ -69,9 +74,9 @@ static size_t at_buf_len;
 static struct k_work cmd_send_work;
 static const char termination[3] = { '\0', '\r', '\n' };
 
-static u8_t uart_rx_buf[UART_RX_LEN];
+static u8_t uart_rx_buf[2][UART_RX_LEN];
+static u8_t *next_buf = uart_rx_buf[1];
 static u8_t *uart_tx_buf;
-static u8_t buf_num;
```

Comment:
Recommend to keep this as the index of current buffer

Reply:
Instead of using multiple chained buffer, I would prefer using double buffer and increasing buffer size if buffer is not enough.